### DISCUSSION TITLE
Marble docs in v6 are referring to v8

### DISCUSSION THREAD

**Discussion by lordrip** [ReactiveX/rxjs#6982] [Report issues other than bug]
Hi there :), if I go to [rxjs v6 docs](https://v6.rxjs.dev/guide/testing/marble-testing), I can see that there's a new helper called `time` which is not present. Giving that I can see
<img width="275" alt="Screenshot 2022-05-30 at 11 56 18" src="https://user-images.githubusercontent.com/97039188/170967930-3256bcb6-890f-4bb8-9a4e-64dab8c41b0e.png">
 I wonder if the link is not correct.

In any case, I'm more than happy to investigate further and submit a PR if needed, but just wanted to know if this is the case

Thanks in advance

**Comment by jakovljevic-mladen** [comment]
It looks like the wrong branch was used while deploying the docs to the V6 machine. I'm gonna try to fix it as soon as possible, and I'm so sorry that it took quite some time to respond. Thanks for reporting.

**Reply by jakovljevic-mladen** [reply]
Hi, this is now fixed and the V6 docs site is now available. Sorry for the inconvenience.
